src/main/java/com/minecolonies/coremod/client/render/RenderFishHook.java

```diff
@@ -90,19 +90,19 @@ public void doRender(@NotNull final EntityFishHook entity, final double x, final
             @NotNull final Vec3d Vec3d = new Vec3d(-0.36D, 0.03D, 0.35D);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)